docker/worker/Dockerfile_arm

```diff
@@ -0,0 +1,33 @@
+#!BuildTag: openqa_worker_arm
```

Comment:
How does this relate to `docker/worker/Dockerfile`? Should that be deleted or replaced?